### ISSUE TITLE
Localized buttons not working for all buttons

### ISSUE THREAD

**Issue by luna404** [WorldHealthOrganization/herams-backend#253]
The localized buttons are only working for the [next button](https://github.com/WorldHealthOrganization/herams-backend/blob/master/docs/media/img/bt_next_en.png)  but not for the others. All others result in image not found. 

![image](https://user-images.githubusercontent.com/18245059/69616587-2a786380-1037-11ea-91f7-4617c4d87f12.png)

```
.. |next| image:: media/img/bt_next.png
   :height: 1.5em
   :alt: next
   :align: top

.. |clear| image:: media/img/bt_clear.png
   :height: 1.5em
   :alt: clear
   :align: top	

````

All buttons are uploaded to the img folder.

![image](https://user-images.githubusercontent.com/18245059/69624333-cb215000-1044-11ea-8a82-9084eeae6ad8.png)

**Comment by SamMousa** [comment]
The format it uses for localized images would be: `bt_next.png ==> bt_next.fr.png`.

**Comment by luna404** [comment]
And this works fine for the next button but not the others.

If you look at the url of the images in the user guide itself there seems to be an issue with the other localized buttons.

http://docs.herams.org/en/latest/_images/bt_next.png --> displays correctly in all languages

https://docs.herams.org/fr/latest/media/img/bt_clear.png --> error image not found

**Comment by SamMousa** [comment]
I think the extra images are confusing you, they don't get removed which is why you might think that they are used. --> I'll update the build script so that old images get removed.

1. For English, the original file name is used, so `bt_next.png` or `bt_clear.png`. 

If you go here: https://github.com/WorldHealthOrganization/herams-i18n/tree/master/docs/media/img you will see that there is no `bt_clear.png`.

2. For `bt_next` you will see there is the base file and the `bt_next.fr.png` file, this is why it works for French. For Arabic, since it didn't find a localized version, it uses `bt_next.png`.

So in conclusion:

- Files ending in `_{language}.png` are ignored, the names should use `.{language}.png`.
- Each image should have a non localized file name for the English version, so `bt_next.png` and NOT `bt_next.en.png`.

**Comment by luna404** [comment]
Well apparently I'm changing the picture in the wrong place which is why i have not picked up on the naming convention.

So just to be clear. Any changes in the base language have to updated in the index file here
https://github.com/WorldHealthOrganization/herams-backend/blob/master/docs/index.rst

But all the the images have to be updated in the herams-i18n repository and not in herams-backend.

**Comment by SamMousa** [comment]
> But all the the images have to be updated in the herams-i18n repository and not in herams-backend.

No, they get copied over there just like the `index.rst` file.
So you upload your media here: https://github.com/WorldHealthOrganization/herams-backend/tree/master/docs/media

And then it gets copied to here:  https://github.com/WorldHealthOrganization/herams-i18n/tree/master/docs/media/img

**Comment by luna404** [comment]
Ok. The part that caused my confusion is that the  bt_next.png does not exists in this folder but only in the herams-i18n folder.

**Comment by SamMousa** [comment]
>  I'll update the build script so that old images get removed.

That's what I'll fix. The problem is that the copying is dumb, so it just copies all new files to the target location but doesn't remove files that no longer exist in the source location!

**Comment by luna404** [comment]
I see. Well issue is fixed. Thanks for your help.
